src/Endpoint/PaymentMethods.php

```diff
@@ -5,7 +5,7 @@
 use \Thawani\RestAPI;
 
 /**
- * Thwawni gateWay Payment endpoint handler
+ * Thwawni gateWay Payment Methods endpoint handler
```

Comment:
```suggestion
 * Thawani gateway Payment Methods endpoint handler
```

Noticed some more typos